lib/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -542,7 +558,7 @@ public void onSuccess(@Nullable final DatabaseUser user) {
 
         @Override
         public void onFailure(@NonNull final AuthenticationException error) {
-            Log.e(TAG, "Failed to create the user: " + error.getMessage(), error);
+            Log.e(TAG, "Failed to create the user: " + error.getCode(), error);
```

Comment:
Changed a few log lines, moved them up in the code and used the error code instead. These might be removed before the release.